packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -0,0 +1,138 @@
+import { buildSelectedNetworkControllerMessenger } from './utils';
+import type { SelectedNetworkControllerOptions } from '../src/SelectedNetworkController';
+import SelectedNetworkController from '../src/SelectedNetworkController';
+
+describe('SelectedNetworkController', () => {
+  it('can be instantiated with default values', () => {
```

Comment:
It looks like some of the tests here are grouped by method, but some are not. What do you think about being consistent? For instance, this one would be grouped under `describe('constructor', ...)`, the next one would be grouped under `describe('setNetworkClientIdForDomain', ...)`, etc.